README.md

```diff
@@ -1,81 +1,59 @@
 # Wagtail shortcode
 
-Draftail Shortcode Entity, adds link-like shortcodes to Draftail.
+Wagtail shortcode adds a custom Draftail enitity to provide a custom linktype ("shortcode"), giving you the ability to assign values in a similar way to normal links as well as decide how they should be rendered on the page.
 
 [![License: BSD-3-Clause](https://img.shields.io/badge/License-BSD--3--Clause-blue.svg)](https://opensource.org/licenses/BSD-3-Clause)
 [![PyPI version](https://badge.fury.io/py/wagtail-shortcode.svg)](https://badge.fury.io/py/wagtail-shortcode)
 [![shortcode CI](https://github.com/wagtail/wagtail-shortcode/actions/workflows/test.yml/badge.svg)](https://github.com/wagtail/wagtail-shortcode/actions/workflows/test.yml)
 
-## Links
-
-- [Documentation](https://github.com/wagtail/wagtail-shortcode/blob/main/README.md)
-- [Changelog](https://github.com/wagtail/wagtail-shortcode/blob/main/CHANGELOG.md)
-- [Contributing](https://github.com/wagtail/wagtail-shortcode/blob/main/CONTRIBUTING.md)
-- [Discussions](https://github.com/wagtail/wagtail-shortcode/discussions)
-- [Security](https://github.com/wagtail/wagtail-shortcode/security)
-
-## Supported versions
-
-- Python ...
-- Django ...
-- Wagtail ...
 
 ## Installation
 
 - `python -m pip install wagtail-shortcode`
 - ...
 
-## Contributing
 
-### Install
+Add 'wagtail_shortcode' to INSTALLED_APPS.
 
-To make changes to this project, first clone this repository:
+Register a [LinkHandler](https://docs.wagtail.org/en/stable/extending/rich_text_internals.html#rewrite-handlers), to control how the new linktype renders on the page.
```

Comment:
Nitpick: this may be used on non-pages. Maybe we could tweak the wording here to eg "is rendered by the |richtext filter" or "renders on the frontend"?

Reply:
Good idea!